::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sun, 26 Mar 2017 20:14:55 +0200
Subject: [PATCH] FIXME: compute.pp: Temporarily disable apparmor

Apparmor profiles need to be refreshed for Armband specific
version of libvirt/qemu/etc.; until updated profiles land,
disable libvirt qemu security driver.

JIRA: https://jira.opnfv.org/browse/ARMBAND-235

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 deployment/puppet/openstack_tasks/manifests/roles/compute.pp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/deployment/puppet/openstack_tasks/manifests/roles/compute.pp b/deployment/puppet/openstack_tasks/manifests/roles/compute.pp
index d81be2a..2844c72 100644
--- a/deployment/puppet/openstack_tasks/manifests/roles/compute.pp
+++ b/deployment/puppet/openstack_tasks/manifests/roles/compute.pp
@@ -538,9 +538,11 @@ class openstack_tasks::roles::compute {
         require => Package['apparmor'],
       }

+      # FIXME(armband): HACK: Disable apparmor until profiles are updated
+      # https://jira.opnfv.org/browse/ARMBAND-235
       file_line { 'qemu_apparmor':
         path    => '/etc/libvirt/qemu.conf',
-        line    => 'security_driver = "apparmor"',
+        line    => 'security_driver = "none"',
         require => [Package['libvirt'], Service['apparmor']],
         notify  => Service['libvirt']
       }
